PE/__init__.py

```diff
@@ -776,7 +817,7 @@ def parseImports(self):
 
                 arrayoff = self.psize * idx
                 if self.filesize is not None and arrayoff > self.filesize:
-                    self.imports = [] # we probably put grabage in  here..
+                    imports_list = [] # we probably put garbage in here
```

Comment:
So this won't actually clear the list you pass in. You'll need to call pop on the list until it's empty.

Reply:
Good point.  Fixed it.

Reply:
I'm not seeing it? At least not on this line, which is the important line to clear things.

The additional clearings on lines 757 and 770 I don't think are necessary (and in fact are a cause of a recursion error in the CI).

My recommendation would be to get rid of the `imports_list` parameter to the parseImportTable function, keep a list local to parseImportTable of the currently parsed imports, and just have the parseImportTable function return that list (or an empty list in case of error) of imports it parsed. That way this line (line 820) can just be a simple `return []`, and you easily sidestep the current recursion error that's happening in the CI. This keeps things fairly simple in terms of not modifying any class member variables or parameters. And then lines 767 and 780 are just assigning imports/delayImports the results of parseImportTable.